provision/kubernetes/kubernetes.tf

```diff
@@ -65,6 +65,10 @@ resource "openstack_networking_floatingip_v2" "floatip_1" {
   pool = "public"
 }
 
+locals {
+  contrail_path = "$HOME/go/src/github.com/Juniper/contrail"
```

Comment:
this should be dependent to branch checked out, otherwise you hav a shared irectory betwen branches and this may cause conflicts

Reply:
At once we only need one branch so there wont be any conflicts. But I've got an idea to add another parameter - ref from gerrit, so there would be git clone and git checkout on the ref, so that we could test changes that has not been merged yet.